scripts/datageneration/property_generator.py

```diff
@@ -0,0 +1,111 @@
+from typing import List
+
+import numpy as np
+from datageneration.data_model import TagAttributeExample, TagAttribute, Property
+
+
+# numerical value generator
+def get_random_decimal_with_metric(range):
+    '''
+    TODO: this should be reworked -- threshold should be defined based on metric
+    '''
+    h_ = np.random.choice(np.arange(range), 1)[0]
+    if np.random.choice([True, False], 1)[0]:
+        h_ = h_ / np.random.choice([10, 100], 1)[0]
+
+    h_ = str(h_) + " " + np.random.choice(["m", "km", "in", "ft", "yd", "mi", "le"], 1)[0]  # "cm",
```

Comment:
If I recall correctly, we agreed that we will continue to train the model to understand CM and MM, and then handle these cases via a front-end popup to tell users that the metrics are invalid, no? Also, the numeric thresholds should be 0 - five digits, independent of the metric.

Reply:
yes, 5 digit version has been not implemented. . I remember meter is the small unit, we discarded cm/mm. Could you enlight us @TilmanDW ?

Reply:
@isspek we agreed to remove the postprocessing of cm and mm. However, the model should detect mm and cm and then we handle it in the frontend